src/ocm/services/apis/ClustersAPI.ts

```diff
@@ -203,6 +205,39 @@ const ClustersAPI = {
       `${ClustersAPI.makeBaseURI()}?ams_subscription_ids=${subscriptionIds.toString()}`,
     );
   },
+
+  getManifests(clusterId: Cluster['id']) {
+    return client.get<ListManifests>(`${ClustersAPI.makeBaseURI(clusterId)}/manifests`);
+  },
+  createCustomManifest(clusterId: Cluster['id'], params: CreateManifestParams) {
+    return client.post<Cluster, AxiosResponse<Cluster>, CreateManifestParams>(
+      `${ClustersAPI.makeBaseURI(clusterId)}/manifests`,
+      params,
+    );
+  },
+  removeCustomManifest(clusterId: Cluster['id'], folderName: string, fileName: string) {
+    let queryParams = '?';
+    queryParams += folderName ? `folder=${folderName}&` : '';
+    queryParams += fileName ? `file_name=${fileName}` : '';
+    return client.delete<void>(`${ClustersAPI.makeBaseURI(clusterId)}/manifests${queryParams}`);
+  },
```

Comment:
If I understand correctly, `folderName` is either "manifest" or "openshift" and it cannot be `undefined` or `''` on our side. And looking at the swagger definition, `file_name` is a required argument. 

So why not just do this?

```suggestion
  removeCustomManifest(clusterId: Cluster['id'], folderName: string, fileName: string) {
    return client.delete<void>(`${ClustersAPI.makeBaseURI(clusterId)}/manifests?folder=${folderName}&file_name=${fileName}`);
  },
```